packages/discord.js/src/structures/interfaces/InteractionResponses.js

```diff
@@ -225,6 +225,14 @@ class InteractionResponses {
     return options.fetchReply ? this.fetchReply() : undefined;
   }
 
+  /**
+   * Indicates whether this interaction can be replied to.
+   * @returns {boolean}
+   */
+  isRepliable() {
```

Comment:
Interaction#isRepliable() is a type guard.

It narrows Interaction to all the Interactions which can be replied.  Like 6ou can't reply to autocomplete Interaction. So it removes it.